### ISSUE TITLE
PartMapper is nearly impossible to read

### ISSUE THREAD

**Issue by Guard13007** [Sujimichi/KerbalX#15]
Dark blue text on a black background... Please use bright colors so it can be read.

![image](https://cloud.githubusercontent.com/assets/15093567/18424445/966fa6a2-7864-11e6-899e-14c0f3a04a35.png)

**Comment by Sujimichi** [comment]
Agreed! They looked much clearer when I tested it, but that looks impossible to read! I will change that and update this issue when I've done it.
